packages/odyssey-react-mui/src/SearchField.tsx

```diff
@@ -30,9 +31,14 @@ import {
 } from "./FieldComponentProps";
 import type { HtmlProps } from "./HtmlProps";
 import { getControlState, useInputValues } from "./inputUtils";
+import { OdysseyThemeProvider } from "./OdysseyThemeProvider";
 
 export const searchVariantValues = ["outline", "filled"] as const;
 
+const StyledContainerToEnsureThemeProviderDivIsFullWidth = styled("div")({
+  width: "100%",
+});
+
```

Comment:
We discussed possible other solutions for this. Not 100% sure the solution, but one is putting `width: "inherit"` on `OdysseyThemeProvider`'s `div`, and another is putting a `flex-basis: "100%"` on this one.